install_config/configuring_ephemeral.adoc

```diff
@@ -0,0 +1,62 @@
+[[install-config-configuring-ephemeral-storage]]
+= Configuring Ephemeral Storage
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+{product-title} can be configured to allow management of ephemeral
+storage of pod and container working data.  While containers
+have been able to utilize writable layers, logs directories, and
+EmptyDir volumes, this storage has been subject to a number of
+limitations,
+xref:../install/architecture/additional_concepts/ephemeral-storage.adoc[as
+discussed here].
+
+Ephemeral storage management permits administrators to limit the
+resources consumed by individual pods and containers, and for pods and
+containers to specify requests and limits on their use of such
+ephemeral storage.  It is a tech preview new in {product-title} 3.10
+and is disabled by default.
+
+Note that this tech preview does not change any of the mechanisms for
+making local storage available in {product-title}; the existing
+mechanisms (root directory or runtime directory) are still used.  This
+preview only provides mechanism for managing the use of this resource.
+
+[[ephemeral-storage-enabling-ephemeral-storage]]
+=== Enable Ephemeral Storage
+Enable the `LocalStorageCapacityIsolation` feature gate on all masters and nodes.
+
+. Edit or create the master configuration file on all masters (*_/etc/origin/master/master-config.yaml_* by default) and add  `LocalStorageCapacityIsolation=true` under the `apiServerArguments` and `controllerArguments` sections:
++
+[source, yaml]
+----
+apiServerArguments:
+   feature-gates:
+   - LocalStorageCapacityIsolation=true
+...
+
+ controllerArguments:
+   feature-gates:
+   - LocalStorageCapacityIsolation=true
+...
+----
+
+. On all nodes, edit or create the node configuration file (*_/etc/origin/node/node-config.yaml_* by default)  and add `LocalStorageCapacityIsolation=true` feature gate under `kubeletArguments`:
```

Comment:
@sjenning @aveshagarwal - this needs to be updated to note the configmap that holds node config in 3.10.

Reply:
Yes, depending upon the node config group a node is in, it would require updating related configmap. For example, if a node is in `node-config-compute` group, it would mean update the node-config-compute ConfigMap in openshift-node namespace.

```
oc edit cm node-config-compute -n openshift-node
```
And edit kubeletArguments as needed.

Reply:
Or if the config for all nodes needs to be updated, that means editing and updating config maps in openshift-node namespace.

/etc/origin/node/node-config.yaml is not updated directly now.

Reply:
@RobertKrawitz @derekwaynecarr Let me know if more information is required here.